src/locale/locales/it/messages.po

```diff
@@ -757,11 +757,11 @@ msgstr "Aspetto"
 
 #: src/view/screens/Settings/index.tsx:475
 msgid "Appearance settings"
-msgstr ""
+msgstr "Aspetto"
```

Comment:
```suggestion
msgstr "Impostazioni dell'aspetto"
```
This is also an `accessibiltyLabel`, so I think making clear that it's related to settings is probably a good idea?